From 9497a6ec97f6d4f26bd95640bf9cf7e197662059 Mon Sep 17 00:00:00 2001
From: Randy Li <randy.li@rock-chips.com>
Date: Mon, 31 Oct 2016 16:28:42 +0800
Subject: [PATCH] video: build rockchip video driver modules independently

There is not need to force the video driver dependency in parent
tree.

Also there would be too many entries in a menu, I create a submenu
for them.

Change-Id: Id3bd54fa6274b1311ff6b20f95ffeb0d1851d9f9
Signed-off-by: Randy Li <randy.li@rock-chips.com>
---
 drivers/video/Kconfig  | 4 ++++
 drivers/video/Makefile | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/drivers/video/Kconfig b/drivers/video/Kconfig
index 1239911ddd0c..bdb38b883762 100644
--- a/drivers/video/Kconfig
+++ b/drivers/video/Kconfig
@@ -34,7 +34,11 @@ source "drivers/video/fbdev/Kconfig"
 endmenu
 
 source "drivers/video/backlight/Kconfig"
+
+menu "Rockchip Misc Video driver"
 source "drivers/video/rockchip/Kconfig"
+endmenu
+
 source "drivers/video/adf/Kconfig"
 
 config VGASTATE
diff --git a/drivers/video/Makefile b/drivers/video/Makefile
index c792897d823f..52ae4ec8f04b 100644
--- a/drivers/video/Makefile
+++ b/drivers/video/Makefile
@@ -5,7 +5,7 @@ obj-$(CONFIG_ADF)		  += adf/
 obj-$(CONFIG_VT)		  += console/
 obj-$(CONFIG_LOGO)		  += logo/
 obj-y				  += backlight/
-obj-$(CONFIG_FB_ROCKCHIP)	  += rockchip/
+obj-y				  += rockchip/
 
 obj-y				  += fbdev/
 
-- 
2.35.3

